### PR TITLE
[FLINK-16102][docs-zh] Translate docs/dev/table/hive/scala_shell_hive.zh.md into Chinese

### PR DESCRIPTION
**Pull request by zhule0601**
## What is the purpose of the change

This pull request translates the scala_shell_hive.zh.md document into Chinese.

## Brief change log

- Translate docs/dev/table/hive/scala_shell_hive.zh.md into Chinese 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency):  no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no 
  - The runtime per-record code paths (performance sensitive):  no 
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no 
  - The S3 file system connector: no 

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

File: docs/dev/table/hive/scala_shell_hive.zh.md [hunk 1]
```diff
@@ -22,18 +22,18 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-NOTE: since blink planner is not well supported in Scala Shell at the moment, it's **NOT** recommended to use Hive connector in Scala Shell.
+注意：目前 blink 计划器还不能很好的支持 Scala Shell，因此 **不** 建议在 Scala Shell 中使用 Hive 连接器。
```
Comment (klion26) [inline docs/dev/table/hive/scala_shell_hive.zh.md:25]: planner 翻译成计划器合理吗？

File: docs/dev/table/hive/scala_shell_hive.zh.md [hunk 2]
```diff
@@ -22,18 +22,18 @@ specific language governing permissions and limitations
...
 
-[Flink Scala Shell]({{ site.baseurl }}/ops/scala_shell.html) is a convenient quick way to try flink. 
-You can use hive in scala shell as well instead of specifying hive dependencies in pom file, packaging your program and submitting it via flink run command.
-In order to use hive connector in scala shell, you need to put the following [hive connector dependencies]({{ site.baseurl }}/dev/table/hive/#depedencies) under lib folder of flink dist .
+[Flink Scala Shell]({{ site.baseurl }}/zh/ops/scala_shell.html) 是快速上手 Flink 的好方法。
+你需要在 pom 中引进 Hive 的依赖，打包并通过 Flink run 来提交 Hive 作业，而在 Scala Shell 中使用 Hive 连接器可以代替这种提交方式。
```
Comment (klion26) [inline docs/dev/table/hive/scala_shell_hive.zh.md:28]: 这句话读起来怪怪的，是不是可以考虑换成类似“你可以在 Scala Shell 中直接使用 Hive 连接器，而不需要在 pom 中添加 Hive 相关依赖，并提交作业” 或者其他类似的

File: docs/dev/table/hive/scala_shell_hive.zh.md [hunk 3]
```diff
@@ -22,18 +22,18 @@ specific language governing permissions and limitations
...
+想要在 Scala Shell 中使用 Hive 连接器，你需要把 [Hive 连接器依赖项]({{ site.baseurl }}/zh/dev/table/hive/#depedencies) 放在 Flink 源码的 lib 文件夹下
```
Comment (klion26) [inline docs/dev/table/hive/scala_shell_hive.zh.md:29]: 比较奇怪的是原文中 “Scala Shell” 的拼写并不统一，有 “Scala Shell” 有 “scala shell”。你觉得哪个更好呢？另外你觉得原文中的描述是否需要进行统一呢？

Comment (klion26) [inline docs/dev/table/hive/scala_shell_hive.zh.md:29]: 这里不是放到 “Flink 源码” 的 lib 文件夹下，而是最后打出来的 dist 包中的 lib 目录下

Comment (zhule0601) [reply docs/dev/table/hive/scala_shell_hive.zh.md]: 我认为这里都是特指 Scala Shell , 应该统一为大写 。另外我把英文的md 也更新了下。

**Comment by klion26** [comment]
@zhule0601 thanks for addressing my comments, LGTM now.
ping @wuchong

**Comment by zhule0601** [comment]
@wuchong Hi, can you help deal with this PR? It has been taking quite a long time.  Orz

**Pull request merged**